### PR TITLE
add koa starter-kit

### PR DESCRIPTION
**Pull request by jerempy**
My contribution for the issue https://github.com/surrealdb/awesome-surreal/issues/35.

The starter-kit: [https://github.com/jerempy/surrealdb-starter-koa](https://github.com/jerempy/surrealdb-starter-koa)

This is a REST API using koa.js and surrealDB as the database.

contains multiple methods on single endpoint with examples of request and responses

### CODE REVIEW

**Comment by kearfy** [comment]
Hello @jerempy! Thanks for your Hacktoberfest contribution, we really appreciate your effort into our community! 

I have one comment about your code, which has to do with these lines:
- https://github.com/jerempy/surrealdb-starter-koa/blob/main/controllers/events.controllers.js#L13
- https://github.com/jerempy/surrealdb-starter-koa/blob/main/controllers/events.controllers.js#L30
- https://github.com/jerempy/surrealdb-starter-koa/blob/main/controllers/events.controllers.js#L39

If I'm not mistaken, a Record ID for any record, also non-event records, can be passed there. Could you make it so that only `event` records can be passed there?

Thanks!

**Comment by naisofly** [comment]
Thank you @jerempy! Could you include a screenshot or GIF of your application running in the README of your project? Thanks!

File: README.md [hunk 1]
```diff
@@ -137,6 +137,7 @@ If you want to contribute to this list, then please read the [contributing guide
 
 ## Starter Kits
 - [SurrealDB + SvelteKit Starter](https://github.com/spinspire/surrealdb-sveltekit-starter) - Jitesh Doshi.
+- [SurrealDB + Koa starter rest api](https://github.com/jerempy/surrealdb-starter-koa) - Jerempy.
```
Comment (kearfy) [inline README.md:140]: Alphabetical ordering :)
```suggestion
- [SurrealDB + Koa starter rest api](https://github.com/jerempy/surrealdb-starter-koa) - Jerempy.
- [SurrealDB + SvelteKit Starter](https://github.com/spinspire/surrealdb-sveltekit-starter) - Jitesh Doshi.
```

**Review by kearfy — APPROVED** [review]

**Comment by naisofly** [comment]
Hey @jerempy 
Thank you for contributing to SurrealDB during Hacktoberfest 2023 🎉
Kindly send an email to [community@surrealdb.com](mailto:community@surrealdb.com) to coordinate the delivery of your SurrealDB swag! 😁

**Pull request merged**